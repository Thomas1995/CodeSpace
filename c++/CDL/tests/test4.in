Testam daacaa imbricate

Vreau un patrat in punctul (1,1) de latura 2
Vreau un patrat in punctul (3,3) de latura 2
Vreau un patrat in punctul (5,5) de latura 2

Daca figura 0 intersecteaza figura 1
Daca figura 0 intersecteaza figura 2
Vreau un poligon cu punctele (0,0), (3,3), (4,4)
Altfel
Vreau un poligon cu punctele (1,1), (3,3), (4,4)
Gata
Altfel
Vreau un poligon cu punctele (2,2), (3,3), (4,4)
Gata boss

Daca figura 0 intersecteaza figura 2

Daca figura 0 intersecteaza figura 2
Vreau un poligon cu punctele (-1,-1), (3,3), (4,4)
Altfel
Vreau un poligon cu punctele (-2,-2), (3,3), (4,4)
Gata

Daca figura 1 intersecteaza figura 2
Vreau un poligon cu punctele (-7,-7), (3,3), (4,4)
Altfel
Vreau un poligon cu punctele (-8,-8), (3,3), (4,4)
Gata

Altfel

Daca figura 1 intersecteaza figura 2
Vreau un poligon cu punctele (-3,-3), (3,3), (4,4)

Daca figura 0 intersecteaza figura 2
Vreau un poligon cu punctele (-9,-9), (3,3), (4,4)
Altfel
Vreau un poligon cu punctele (-10,-10), (3,3), (4,4)
Gata

Vreau un poligon cu punctele (-3,-3), (3,3), (4,4)

Altfel
Vreau un poligon cu punctele (-4,-4), (3,3), (4,4)
Gata

Daca figura 0 intersecteaza figura 2
Vreau un poligon cu punctele (-5,-5), (3,3), (4,4)
Altfel
Vreau un poligon cu punctele (-6,-6), (3,3), (4,4)
Gata

Gata boss
