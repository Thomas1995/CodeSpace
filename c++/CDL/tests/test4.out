0 Patrat:
1 1
1 3
3 1
3 3
1 Patrat:
3 3
3 5
5 3
5 5
2 Patrat:
5 5
5 7
7 5
7 7
3 Poligon:
1 1
3 3
4 4
4 Poligon:
-3 -3
3 3
4 4
5 Poligon:
-10 -10
3 3
4 4
6 Poligon:
-3 -3
3 3
4 4
7 Poligon:
-6 -6
3 3
4 4
